src/mcts/search.cc

```diff
@@ -192,11 +192,13 @@ int64_t Search::GetTimeToDeadline() const {
 }
 
 namespace {
-inline float GetFpu(const SearchParams& params, Node* node, bool is_root_node) {
+inline float GetFpu(const SearchParams& params, Node* node,
+                    bool is_root_node, bool logit_q = false) {
   const auto value = params.GetFpuValue(is_root_node);
+  const auto Q = logit_q ? node->GetQ() : FastLogit(node->GetQ());
```

Comment:
Why is this opposite of the one in node.h?

Reply:
Wow. I don't think it should be.

Reply:
I think this one is the one that should be flipped